src/NServiceBus.RavenDB.AcceptanceTests/Config.cs

```diff
@@ -29,4 +39,14 @@ public void Cleanup()
 
         Console.WriteLine("Deleted '{0}' database", documentStore.DefaultDatabase);
     }
+
+    public static DocumentStore GetDefaultDocumentStore(ReadOnlySettings settings)
```

Comment:
For me the ATT should be as close as possible to the end user code. It is another way of verifying and live documenting the API. The cast in this case is a necessary evil for the assertions